tests/test_count_motifs.py

```diff
@@ -0,0 +1,104 @@
+"""Unit tests for infer_single_paired.py"""
```

Comment:
The tests are really nice! Well done!  :)

(though probably need some changes after addressing the comments above, sorry about that)